src/org/labkey/test/tests/ApiKeyTest.java

```diff
@@ -145,12 +150,17 @@ private void verifyValidAPIKey(String apiKey, boolean basicAuth) throws IOExcept
         {
             throw new RuntimeException("Response: " + e.getStatusCode(), e);
         }
+        return cn;
     }
 
-    private void verifyInvalidAPIKey(String apiKey) throws IOException
+    private void verifyInvalidAPIKey(String apiKey, @Nullable Connection connection) throws IOException
     {
         boolean isSessionKey = !apiKey.startsWith(API_USERNAME);
```

Comment:
This is no longer a valid check... API keys and session keys no longer start with these prefixes. You can't distinguish them just by looking at them.